cmd/bnsd/scenarios/bnsdtest/bnsdtest.go

```diff
@@ -237,6 +256,28 @@ func initGenesis(t testing.TB, env *EnvConf, filename string) {
 				Admin: weave.Condition("multisig/usage/0000000000000001").Address(),
 			},
 		},
+		"governance": dict{
+			"electorate": []interface{}{
+				dict{
+					"title":    "first electorate",
+					"admin":    env.governance.electors[0],
```

Comment:
Is this an address? Must be so not an ID

Reply:
This is an address.